source/docs/contributing/watching-changes.rst

```diff
@@ -0,0 +1,44 @@
+# Watching changes
+
+This document describes different ways to stay up-to-date on the changes being made to the WPILib suite.
+
+## Stages of development
+
+To provide context about the different ways to stay up to date, below are the relevant stages of development:
+
+- Discord: The [Unofficial FIRST Robotics Competition Discord Server](https://discord.com/invite/frc) contains a programming discussion channel where people bring up their initial ideas and then receive feedback on them, either fleshing them out or learning why they might not be good ideas. Most of the major changes will have some discussion in here, but not all.
```

Comment:
I think it's important to mention:
- the number of major wpilib contributors that are active (and if this channel is considered the preferred place for ideas to being
- The reason the discord works i.e instant messaging makes the conversations move quickly, allowing ideas to get out of the idea phase and into the planning stage quicker

Reply:
> * (and if this channel is considered the preferred place for ideas to being

Could you finish this thought?